packages/desktop-gui/src/project/project-model.js

```diff
@@ -72,8 +72,11 @@ export default class Project {
   @computed get displayName () {
     if (this.name) return this.name
 
-    let splitName = _.last(this.path.split('/'))
-    return _.truncate(splitName, { length: 60 })
+    // need normalize windows paths with \ before split
+    const normalizedPath = this.path.replace(/\\/g, '/')
```

Comment:
This is unnecessary because `projectName` is given to you via the project's configuration. You don't need to calculate it yourself and this keeps the logic centralized in the server code and automatically takes into account different OS's like windows vs linux vs mac.